cmd/cloud-network-config-controller/main.go

```diff
@@ -297,7 +299,7 @@ func createFeatureGateAccessor(ctx context.Context, cfg *rest.Config, operatorNa
 		Kind:       "Deployment",
 		Namespace:  namespace,
 		Name:       deploymentName,
-	})
+	}, passiveClock)
```

Comment:
I don't think this will work, passiveClock is an empty interface so when CNCC will try to generate an event the code will likely panic.
To solve this you could just use `clock.RealClock`:
```suggestion
	}, clock.RealClock{})
```